cmd/autoowners/main.go

```diff
@@ -81,7 +81,7 @@ func loadRepos(configRootDir string, blocked blocklist, configSubDirs, extraDirs
 				logrus.WithField("org", org).Info("org is on organization blocklist, skipping")
 				continue
 			}
-			repoDirs, err := os.ReadDir(filepath.Join(subdirectory, orgDir.Name()))
+			repoDirs, err := os.ReadDir(filepath.Clean(filepath.Join(subdirectory, orgDir.Name())))
```

Comment:
Doesn't `Join` already perform a `Clean` internally? I guess the snyk is unaware of that?

Reply:
Right! I will remove it, talk with Snyk guys and see what happens.

https://cs.opensource.google/go/go/+/refs/tags/go1.23.4:src/path/filepath/path.go;l=130 
https://cs.opensource.google/go/go/+/refs/tags/go1.23.4:src/path/filepath/path_unix.go;l=36